src/main/ts/components/Editor.ts

```diff
@@ -28,10 +28,12 @@ const renderIframe = (ce: any, id: string, elementRef: Ref<Element | null>) =>
     ref: elementRef
   });
 
+const defaultInitVaues = { selector: undefined, target: undefined };
```

Comment:
It looks good to me but there is a typo in the variable name
```suggestion
const defaultInitValues = { selector: undefined, target: undefined };
```

Reply:
fixed, thanks :)